services/QuillCMS/app/workers/rematch_response_worker.rb

```diff
@@ -21,13 +21,17 @@ class RematchResponseWorker
 
   class LambdaHTTPError < StandardError; end
 
-  def perform(response_id, question_type, question_hash, reference_response_ids)
+  def perform(response_id, question_type, question_hash, reference_response_ids, options)
     response = Response.find_by(id: response_id)
     return unless response
 
     reference_responses = Response.where(id: reference_response_ids).to_a
 
     rematch_response(response, question_type, question_hash, reference_responses)
+
+    return unless options['fire_pusher_alert']
```

Comment:
changing this line to 

`return unless options['fire_pusher_alert'] && options['question_key']`

removes an edge case where we inadvertently send Pusher events with a null question ID